bundles/org.openhab.binding.withings/README.md

```diff
@@ -0,0 +1,57 @@
+# Withings Binding
+
+The Withings binding uses the official public Withings data API. It integrates the following Withings product types including related measures:
+
+- Scale
+- Sleep Monitor
+
+Measured data of the user is combined in a "Person" thing.
+
+There is only 1 user/person available per Withings account (that is limited by the Withings data API). You can choose the user when you generate the auth-code, see Binding Configuration.
+
+The channels are updated every 5 minutes with new data (when available). Therefore the binding causes network traffic every few minutes to synchronize the data (it transfers data from and to the official public Withings data API).
+
+## Binding Configuration
+
+### 1. Application Creation
+
+Register a new Withings application here: https://account.withings.com/partner/add_oauth2

```

Comment:
How can I create an application there? What is the Callback URL? Everyone needs to register openHAB as an application at this site?